### ISSUE TITLE
Not working with Sketch 53.1

### ISSUE THREAD

**Issue by jpanfilova** [getflourish/Sketch-Style-Inventory#103]
It generates a new page, but no elements on it, the artboard stays empty.

**Comment by mdominguez** [comment]
Same here

**Comment by steadynow** [comment]
Same issue here.

**Comment by Raghav789** [comment]
same here

**Comment by FabienLaborie** [comment]
Up!

**Comment by moniquep** [comment]
Yeah, is it time this plugin is laid to rest? It's been like this for a while now.

**Comment by abbrechen** [comment]
Same problem with Sketch 53.2

**Comment by urbantrout** [comment]
Is there an alternative plugin?

**Comment by FabienLaborie** [comment]
Yes, there is an alternative. Use Automate Sketch, with the Style > Create
typography guide function.
—
*Fabien Laborie*
Product and Branding Designer, Founder of @SketchMTL
<https://www.facebook.com/sketchmtl/> — Join us on Slack
<https://sketchmtl.now.sh/>.
—
(+1) 514 994-2540
www.fabienlaborie.ca
—

**Comment by getflourish** [comment]
Here’s an alternative for generating color inventories:
https://github.com/themainingredient/colormate
